.travis.yml

```diff
@@ -138,10 +67,14 @@ before_script:
 
 script:
   - export SEQAN3_NO_VERSION_CHECK=1
+  - export CCACHE_COMPRESS=true
+  - export CCACHE_COMPRESSLEVEL=6
+  - export CCACHE_MAXSIZE=5G
```

Comment:
this is now enough or not important because it is a nightly build?

Reply:
When we compress it, it gets considerably smaller, I think the GitHub actions ccaches are around 30-60MiB, so we should have always compressed them; 5G are now more than enough